remote-method/USAGE

```diff
@@ -0,0 +1,6 @@
+Description:
+  Creates a new remote method in the LoopBack application.
+
+Example:
+
+  slc loopback:remote-method
```

Comment:
> `slc loopback:remote-method`

I am proposing to shorten this to `slc loopback:method` to make the command shorter to type.

@ritch @superkhau thoughts?